x/emissions/keeper/keeper.go

```diff
@@ -4597,3 +4603,77 @@ func (k Keeper) SetLatestForecasterWeight(ctx context.Context, topicId TopicId,
 	}
 	return k.latestForecasterWeights.Set(ctx, collections.Join(topicId, worker), weight)
 }
+
+// GetMonthlyReputerRewards retrieves the total reward for the month going to reputers.
+func (k *Keeper) GetMonthlyReputerRewards(ctx context.Context) (cosmosMath.Int, error) {
```

Comment:
In all their usage, reputer and topic rewards are used together, and updated together, what do you think about exposing only set of funcs (e.g. `SetMonthlyRewards(ctx, reputer, topic)`) for both of them?

This would ensure by design they are updated together, as they should